### DISCUSSION TITLE
Welcome to wrf_hydro_gis_preprocessor Discussions!

### DISCUSSION THREAD

**Discussion by kmsampson** [NCAR/wrf_hydro_gis_preprocessor#4] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mary-chen** [comment]
when i run the script, there's sth. wrong about the module. if possible, could u help me?

(wrfh_gis_env) E:\acer\Desktop\wrf_hydro_gis_preprocessor-master\wrfhydro_gis>python Create_Domain_Boundary_Shapefile.py -h
Traceback (most recent call last):
  File "Create_Domain_Boundary_Shapefile.py", line 34, in <module>
    from wrfhydro_functions import WRF_Hydro_Grid
  File "E:\acer\cx\Desktop\wrf_hydro_gis_preprocessor-master\wrfhydro_gis\wrfhydro_functions.py", line 44, in <module>
    import osr
ModuleNotFoundError: No module named 'osr'
